### PR TITLE
fix(content): Minor HR fixes and player experience edits

### PR DESCRIPTION
**Pull request by MasterOfGrey**
This simply addresses a number of minor fixes for HR.

(Updated 7/2/2023)
- An old log entry.
- An inconsistent name spelling.
- Relocates an event to the appropriate spot.
- Removes the random spawns from Elenchus for balance (alternative will be added by another PR - see #8316 )
- Fixes issues with the geocoris sequence where text spawns out of order, and where it was possible to get locked from progressing.
- Adds choices and fixes text to make it clearer that defeating the Lich is the main goal, and saving the Geocoris is a side-goal.
- Adds a pointer mission for a spot people get confused at.
- Updates out-of-date comments.
- Adds an `on fail` dialogue for the critical mission.
- Extends an event timer to more realistically extend beyond the narrative.

I was meant to add to this but I just haven't had time so I'm getting these out there sooner than later before I have to deal with even more merge conflicts than it already had..
(Side note: how have we merged 270 PR's in the last 2 months??)

### CODE REVIEW

**Comment by MasterOfGrey** [comment]
Hang on, adding to this.

**Comment by MasterOfGrey** [comment]
@Zitchas thank you for reminding me to get onto this change.

**Comment by Zitchas** [comment]
You are welcome. 

One thought: the mission description does note that the fleet will fight other ships that arrive, just not the planet itself. Maybe there should be a couple pirate NPCs spawned somewhere at that point in the mission that will take one and two minutes respectively to arrive so that the escorts *are* fighting stuff now and then.

**Review by EjoThims — APPROVED** [review]

**Comment by MasterOfGrey** [comment]
> One thought: the mission description does note that the fleet will fight other ships that arrive, just not the planet itself. Maybe there should be a couple pirate NPCs spawned somewhere at that point in the mission that will take one and two minutes respectively to arrive so that the escorts _are_ fighting stuff now and then.

I'm not aware of a way of doing this without there being some kind of `wait` or `loiter` function for NPC's. At the moment any NPC's added would wither come and hunt you down on your way there, or have to be in the system already when you get there.

File: data/hai/hai reveal 5 philosophers.txt [hunk 1]
```diff
@@ -964,7 +964,7 @@ mission "Hai Reveal [B11] Moving Together"
 			action
 				set "Hai Reveal [B11] Moving Together: label secret"
 			`	Remington explains. "This operation has gone on long enough that the popular opinion is that the Hai must obviously exist, even though there are no explicit voices saying as such. It has in fact almost become a joke; the perception being that, 'The Navy are blockading the Hai to hide humanity from them, because the blockade isn't doing a good job of being inconspicuous.' There have been memes of the Navy hiding their ships behind far too narrow light poles while they distract a squirrel with their headlights. Now it's a matter of being able to present good news. Place a successful spin on things."`
-			`	"Our problem," says Khaine. "Is Elenchus."`
+			`	"Our problem," says Kaine. "Is Elenchus."`
```
Comment (warp-core) [inline data/hai/hai reveal 5 philosophers.txt:967]: Should the second part not follow on from the previous part as a single sentence, so a comma there instead of a full stop and no capital `is`?

Comment (MasterOfGrey) [reply data/hai/hai reveal 5 philosophers.txt:967]: That is how I was taught grammar, but I keep being corrected cause apparently it's not standard for US?
That's an EJo/Road/Wiz question at the end of the day.

Comment (bene-dictator) [reply data/hai/hai reveal 5 philosophers.txt:967]: There shouldn't be a full stop there. Kaine isn't actually saying it like "Our problem. Is Elenchus." I'd imagine.

Not sure why it's different across the ditch : )

Comment (EjoThims) [reply data/hai/hai reveal 5 philosophers.txt:967]: Warp's got it right, yea.

File: data/hai/hai reveal 5 philosophers.txt [hunk 2]
```diff
@@ -1233,11 +1233,16 @@ fleet "Elenchus Tribute"
...
+# They suspect you're coming so they're on alert, and non-essential ships aren't flying about.
+# Also, removing the warship spawn stops your NPC escorts from getting overwhelmed inappropriately.
+event "elenchus can be dominated"
 	planet "Elenchus"
 		tribute 5000
 			threshold 2400
 			fleet "Elenchus Tribute"
+	system "Devil-Hide"
+		remove fleet "Devil-Run Warships"
+		remove fleet "Devil-Run Miners"
```
Comment (Hurleveur) [inline data/hai/hai reveal 5 philosophers.txt:1245]: just doing this by itself causes a huge plothole if you ask me
like, why dont they help with the tribute fleet? earlier dialog by Choot'k mentioned they'd cover you from incoming ships, too, I think?
I'd move this to another PR where we would replace it by smth else

**Review by Hurleveur — COMMENTED** [review]
I'd move this one thing to another PR so we can just merge this as is

**Review by EjoThims — CHANGES_REQUESTED** [review]
Forgot to hit the button again.

File: data/hai/hai reveal 1 intro.txt [hunk 3]
```diff
@@ -793,7 +793,7 @@ mission "Hai Leaks Response 4"
 		dialog `You have returned, but the fate of the pirates and the captured Hai is still in doubt. Ensure that the Black Lich is defeated and that the fate of the captured Hai, whether deceased or safely with you, is certainly known.`
 	to complete
 		or
-			has "00 Hai Leaks Response 4B: declined"
+			has "00 Hai Leaks Response 4B: offered"
```
Comment (warp-core) [inline data/hai/hai reveal 1 intro.txt:796]:
```suggestion
			has "ship model (all): Geodesic Geocoris"
```

**Review by Hurleveur — APPROVED** [review]
since there is another PR with the defense cannon we dont need the fleet anymore indeed, so may as well remove it here
the fixes look good

File: data/hai/hai reveal 1 intro.txt [hunk 4]
```diff
@@ -792,24 +796,24 @@ mission "Hai Leaks Response 4"
...
 			`	He sighs deeply. "It is a shame though that the kidnapped Hai could not be saved, but the other Hai captains tell me it was a mess out there."`
 			`	In an oddly human gesture that feels very deliberate, he places one of his furry hands comfortingly on your shoulder, "I'm sure you did all you could."`
 				goto end
 			label "saved geocoris"
-			`	"Your rescue of our kidnapped people is deeply appreciated also. Lucky for you, the merchant who owned the Geocoris has already filed the insurance claim, so you get to keep the ship as a trophy.`
+			`You have defeated the Black Lich and also successfully reclaimed the Geocoris, and in doing so rescued the kidnapped Hai! On your way back to <planet> the crew you sent across to fly it home reported that something is different about it... It appears that it has been modified with systems of some kind, which are not of Hai design. It also appears that perhaps not everything about those systems is working wholly as intended.`
```
Comment (bene-dictator) [inline data/hai/hai reveal 1 intro.txt:814]: I wonder if this might be better placed in a small "on enter" conversation when you fly into Allhome's system with the Geodesic Geocoris?

It feels out of place for it to be shown when you land, with dialogue coming up next.

File: data/hai/hai reveal 1 intro.txt [hunk 5]
```diff
@@ -792,24 +796,24 @@ mission "Hai Leaks Response 4"
...
+			`	As you disembark you spot the Hai quartermaster checking on the Hai disembarking from the Geocoris, and after a short while he approaches you. "Captain <last> I cannot thank you enough for your rescue of our kidnapped people. You're a hero to them. The Allhome government has also authorized a reward of <payment> for helping to route the pirate problem," he explains while handing you the credits.`
```
Comment (bene-dictator) [inline data/hai/hai reveal 1 intro.txt:815]:
```suggestion
			`	As you disembark you spot the Hai quartermaster checking on the Hai disembarking from the Geocoris, and after a short while he approaches you. "Captain <last>, I cannot thank you enough for your rescue of our kidnapped people. You're a hero to them. The Allhome government has also authorized a reward of <payment> for helping to relieve the pirate problem," he explains while handing you the credits.`
```
Is 'route' the right word to use here? The Oxford Dictionary seems to disagree. I think 'rout' or 'relieve' might be better.

**Review by bene-dictator — CHANGES_REQUESTED** [review]

File: data/hai/hai reveal 1 intro.txt (hunk 4 above)
Comment (MasterOfGrey) [reply data/hai/hai reveal 1 intro.txt:814]: It can't be made to appear there *only* when you've actually got it though, if we could have conditional `on enter`'s then it would be there

Comment (MasterOfGrey) [reply data/hai/hai reveal 1 intro.txt:814]: Also this is where it always triggered/was meant to trigger - it just used to be separate and would trigger independently

File: data/hai/hai reveal 1 intro.txt [hunk 6]
```diff
@@ -841,11 +847,11 @@ event "hai ladybug available"
...
 
 # It is impossible to implement a "capture and return npc" mission.
-# Once the ship is captured, the NPC is gone and the mission is failed.
-# The NPC has an outfit used to determine if you managed to get it back safely.
+# Once the ship is captured, the mission is completed.
+# The NPC is a special ship model used to determine if you managed to get it back safely.
 mission "Hai Leaks Response 4A"
 	name "Recover the Geocoris"
-	description "Recover the stolen Geocoris supporting the pirates in Waypoint, and return to Allhome."
+	description "If possible, recover the stolen Geocoris supporting the pirates in Waypoint, and return it to Allhome."
```
Comment (EjoThims) [inline data/hai/hai reveal 1 intro.txt:854]:
```suggestion
	description "If possible, recover the stolen Geocoris supporting the pirates in Waypoint and return it to Allhome."
```

File: data/hai/hai reveal 6 end.txt [hunk 7]
```diff
@@ -26,8 +26,7 @@ mission "Hai Reveal [C01] Cleanup"
 	on offer
 		event "mountaintop ascendant"
 		fail "Hai Reveal [B12-A] Devil-Hide Fleet"
-		log `Returned the Hai to their own space to bring an end to the missing Hai saga. All that remains is to work out what to do about the leak and the cover story, so thin as to provide no modesty to its wearers.`
-		log `Teeneep and General Choot'k, prominent leaders of the Hai and Unfettered, kissed in a romantic goodbye. They spoke of a future peaceful unification of the Hai.`
+		log `Returned the Hai to their own space to bring an end to the missing Hai saga. Choot'k tries but fails to recruit Teeneep to the Unfettered. They spoke of a future peaceful unification of the Hai, though differ on the specifics despite their closeness. There is one last trip to do for official matters.`
```
Comment (EjoThims) [inline data/hai/hai reveal 6 end.txt:29]:
```suggestion
		log `Returned the Hai to their own space to bring an end to the missing Hai saga. Choot'k tried, unsuccessfully, to recruit Teeneep to the Unfettered. They spoke of a future peaceful unification of the Hai, though differ on the specifics despite their closeness. There is one last trip to do for official matters.`
```

Tense mismatch here, but also the style suggestion versus the repetitive fix.

File: data/hai/hai reveal 1 intro.txt (hunk 4 above)
Comment (EjoThims) [inline data/hai/hai reveal 1 intro.txt:814]:
```suggestion
			`You have defeated the Black Lich, successfully reclaimed the Geocoris, and, in doing so, rescued the kidnapped Hai! On your way back to <planet> the crew you sent across to fly it home reported that something is different about it... It appears that it has been modified with systems of some kind, which are not of Hai design. It also appears that perhaps not everything about those systems is working wholly as intended.`
```

Preferred.

Comment (EjoThims) [inline data/hai/hai reveal 1 intro.txt:814]:
```suggestion
			`You have defeated the Black Lich and also successfully reclaimed the Geocoris and, in doing so, rescued the kidnapped Hai! On your way back to <planet> the crew you sent across to fly it home reported that something is different about it... It appears that it has been modified with systems of some kind, which are not of Hai design. It also appears that perhaps not everything about those systems is working wholly as intended.`
```
Less correct alternative if the style is important for emphasis/pacing.

**Review by EjoThims — CHANGES_REQUESTED** [review]

File: data/hai/hai reveal 1 intro.txt (hunk 4 above)
Comment (MasterOfGrey) [reply data/hai/hai reveal 1 intro.txt:814]: Yeah I'll go with this, because I'm leaning into the "and *also* you did the thing" with the geocoris, since many, many people seem to have thought that was the primary goal.

**Review by EjoThims — APPROVED** [review]

File: data/hai/hai reveal 1 intro.txt [hunk 8]
```diff
@@ -792,24 +796,24 @@ mission "Hai Leaks Response 4"
 	on visit
 		dialog `You have returned, but the fate of the pirates and the captured Hai is still in doubt. Ensure that the Black Lich is defeated and that the fate of the captured Hai, whether deceased or safely with you, is certainly known.`
 	to complete
-		or
-			has "00 Hai Leaks Response 4B: declined"
-			has "Hai Leaks Response 4A: failed"
+		not "Hai Leaks Response 4A: active"
```
[the remark this answers is not in the document]
Comment (Hurleveur) [reply data/hai/hai reveal 1 intro.txt:799]: this does mean it can trigger before it's offered whereas before it wasnt the case

[the remark this answers is not in the document]
Comment (MasterOfGrey) [reply data/hai/hai reveal 1 intro.txt:799]: Granted, I did not, but these two missions have identical offer conditions, so there's no possible way for this to be able to complete before the other one offers.

But now there's no text in 4B, so it doesn't have to juggle the ordering with that, and 4A is an NPC capture mission, so it should only ever be active if the NPC is still alive and independent.
If it's dead or captured, it'd be no longer active, which is the condition for moving forward.
